moped-editor/src/views/projects/projectView/ProjectFunding/ProjectFundingTable.js

```diff
@@ -245,12 +244,6 @@ const ProjectFundingTable = () => {
   const apiRef = useGridApiRef();
   const classes = useStyles();
 
-  /**
-   * User Hook
-   * @type {object} CognitoUserSession
-   */
-  const { user } = useUser();
```

Comment:
This user data was retrieved to be passed to the SubprojectFundingModal component where it was no longer used.